docs/section11/Rule_11-10.md

```diff
@@ -0,0 +1,221 @@
+        - check if the capacity <= 105,000 btu/hr: `elif input_capacity <= 105000:`
+          - the capacity per volume should be < 4000 btu/hr/gallon: `if capacity_per_volume >= 4000:`
+            - add the following text to the note: "For Gas Storage Water heaters, the capacity per volume should be less than 4000 btu/hr/gallon": `rule_note += "For Gas Storage Water heaters, the capacity per volume should be less than 4000 btu/hr/gallon.  "`
+            - set has_fault to true: `has_fault = true`
+          - the volume should be <= 120 gallons: `if storage_volume > 120:`
+            - add the following text to the note: "For Gas Storage Water heaters with capacity less than 105,000 btu/hr, the storage capacity is expected to be less than 120 gallons": `rule_note += "For Gas Storage Water heaters with capacity less than 105,000 btu/hr, the storage capacity is expected to be less than 120 gallons.  "`
+            - set has_fault to true: `has_fault = true`
+          - the supply water temperature should be <= 180F: `if shw_eq.setpoint_temperature > 180:`
+            - add the following text to the note: "For Gas Storage Water heaters with capacity less than 105,000 btu/hr, the supply water temperature is expected to be <= 180F": `rule_note += "For Gas Storage Water heaters with capacity less than 105,000 btu/hr, the supply water temperature is expected to be <= 180F.  "`
+            - set has_fault to true: `has_fault = true`
+          - calculate the expected efficiency for VERY_SMALL draw pattern: `if draw_pattern == "VERY_SMALL": expected_UEF = 0.2674 – (0.0009 * volume)`
+          - calculate the expected efficiency for LOW draw pattern: `if draw_pattern == "LOW": expected_UEF = 0.5362 – (0.0012 * volume)`
+          - calculate the expected efficiency for MEDIUM draw pattern: `if draw_pattern == "MEDIUM": expected_UEF = 0.6002 – (0.0011 * volume)`
+          - calculate the expected efficiency for HIGH draw pattern: `if draw_pattern == "HIGH": expected_UEF = 0.6597 – (0.0009 * volume)`
+          - check whether the uniform_energy_factor is valid: `if uniform_energy_factor != NULL
+            - check if the actual uniform_energy_factor is >= to the expected: `if uniform_energy_factor >= expected_UEF && !has_faults:`
+              - set rule_status to "PASS": `rule_status = "PASS"`
+          - otherwise, set rule status to UNDETERMINED and provide a note: `else: rule_status = "UNDETERMINED"; rule_note = "Uniform Energy Factor is required but not provided"`
+        - otherwise the capacity is > 105,000 btu/hr: `else:`
+          - the capacity per volume should be < 4000 btu/hr/gallon: `if capacity_per_volume >= 4000:`
+            - add the following text to the note: "For Gas Storage Water heaters, the capacity per volume should be less than 4000 btu/hr/gallon": `rule_note += "For Gas Storage Water heaters, the capacity per volume should be less than 4000 btu/hr/gallon.  "`
```

Comment:
Suggest structuring the note similar to suggestions above with references to Table 7.8 and Appendix F so that the reviewer understands the context what this note means.